schemas/com.snowplowanalytics.snowplow/media_player_quality/jsonschema/1-0-0

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity that is tracked with media player events on data quality change",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_quality",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "bitrate": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "The current bitrate in bps.",
+      "minimum": 0,
+      "maximum": 9007199254740991
+    },
+    "droppedFrames": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "The number of dropped frames in the stream.",
```

Comment:
Maybe just specify if this is cumulative or since the last event?

Reply:
Specified since last report as that would be my preference (happy to change if there are other opinions).